fast64_internal/f3d/f3d_writer.py

```diff
@@ -1,3 +1,4 @@
+from email.mime import image
```

Comment:
I feel like this is added by auto-completion

Reply:
(nothing else to add, I just wanted to check if the imports have anything weird)

Reply:
I swear I deleted this. Why does this keep coming back lmao

Reply:
sometimes I hate auto-completion lmao